Paltineanu Rares-Mihai/3.7 - curs4 switch/main.c

```diff
@@ -0,0 +1,112 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+
+int verif(char hex[200])
+{
+    int i = strlen(hex) - 1;
+    for(int j = 0; j <= i; j++)
+    {
+        if(hex[j]>'F')
+        {
+            return 0;
+        }
+
+    }
+    return 1;
+}
+
+///firstly i write the function like this: void convert(char hex[200], double &a)
+///but the error is saying "expected ';', ',', or ')' before '&' token and idk what is meaning
```

Comment:
Well, if you want to pass an address as a parameter, you should be using the `*` operator, like:
` int convert(char *hex, double *a)`
in this case, `char *hex` is equivalent to `char hex[200]`
But I'm not particularly sure what you wanted to write there

Reply:
I just wanted to modify the address of a in the function

Reply:
I wanted to modify the address of a in the function